app/src/main/java/com/mobiledevelopmentfoodapp/MainActivity.kt

```diff
@@ -54,6 +54,10 @@ fun Header(name: String) {
 
 @Composable
 fun ButtonBar() {
+    var cartPopupVisibility by remember{mutableStateOf(false)}
+    //following line of code should be replaced with list of items the user has added to their cart
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.